src/tables/css/tables.css

```diff
@@ -30,6 +30,7 @@ there's a rowspan on the first cell. Case added to the tests. issue#432 */
     border-left-width: 0;
 }
 
+.pure-table tfoot,
```

Comment:
I closed a similar PR for this because I was afraid of introducing these styles to the footer. Is it common for the table footer to have the same style as the `thead`?